snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snykmonitors.charts.helm.k8s.io.crd.yaml

```diff
@@ -10,12 +10,42 @@ spec:
     plural: snykmonitors
     singular: snykmonitor
   scope: Namespaced
-  subresources:
-    status: {}
   validation:
     openAPIV3Schema:
       type: object
-      x-kubernetes-preserve-unknown-fields: true
+      properties:
```

Comment:
This actually now adds validation for the CRD 😵 Would you prefer it's also in its own separate commit?

Reply:
is it part of the `chore: improve Operator description`? If not, I would prefer each code change to be in a separate commit. not a blocker tho :)

Reply:
You've got a point; let's fix this!